src/histolab/slide.py

```diff
@@ -457,11 +462,17 @@ def __iter__(self) -> Iterator[Slide]:
         -------
         generator of `Slide` objects.
         """
+        slide_names = [
+            _name
```

Comment:
I don't understand the need of the underscore before the name. I'd use name without any special characters.

Reply:
@ernestoarbitrio I agree. I just retained the variable name from the original version of the file. OK, removed the underscore.